Resources/doc/security.md

```diff
@@ -0,0 +1,10 @@
+# Security
+
+To completely disable some operations from your application, refers to [Disabling operations](operations.md#disabling-operations).
+
+This bundle does not have custom logic for addressing security concerns. If you wish to do so, you have to rely on the features provided by Symfony. For instance, if you wish to restrict the access of some endpoints, you can use [Access controls directives](http://symfony.com/doc/current/book/security.html#securing-url-patterns-access-control).
+
+It is also possible to use the [Event system](the-event-system.md) for more advanced logic or even [Custom controlers](controllers.md#using-a-custom-controller) if you really need to.
```

Comment:
Event system as changed a lot in #191. It is now more powerful, especially for security handling (`kernel.request` event) and cache (`kernel.response` event). It would be nice to mention how to use the new system (the old one will be removed today).